core/data/stories.md

```diff
@@ -319,7 +319,7 @@
   - utter_try_again_later
   - action_qa_goodbye
 
-## tested positive - no symptoms tested less than 14 days
+## tested positive - no symptoms tested less than 14 days error
```

Comment:
I allowed myself to "re-purpose" this story because it already had a twin for "no symptoms tested less than 14 days" but with a different continuation